### ISSUE TITLE
Anvils do not respect creative mode

### ISSUE THREAD

**Issue by Leomelonseeds** [GC-spigot/AdvancedEnchantments#4273]
### Describe the bug

In vanilla, creative mode users are able to bypass the 40 level limit set on anvil repairs. AdvancedEnchantments seems to have broken this, not allowing creative mode repairs that would cost above 40 levels.

### How to reproduce

Simply attempt to enchant a sword too many times in creative mode. A good way to do this is take an enchant such as Sharpness, and apply books with levels 1, 1, 2, 3, 4, 5 in that order, and repeat with other enchants. Eventually, repairs will no longer be possible, even in creative mode.

### Screenshots / Videos

Expected behavior (vanilla singleplayer):
![image](https://github.com/GC-spigot/AdvancedEnchantments/assets/17891302/1be18956-6480-4206-87f0-df0167dec56b)

AdvancedEnchantments behavior:
![image](https://github.com/GC-spigot/AdvancedEnchantments/assets/17891302/390be89f-92fc-4ddf-92f1-f503f5be7a50)

### "/ae plinfo" link

https://paste.md-5.net/apagacukil

**Comment by SirJben** [comment]
First things first, why would you let your survival players get creative mode?

**Comment by Leomelonseeds** [comment]
I'm not letting survival players get creative mode. I made this issue because I'd like the plugin to have good parity with vanilla. As a server admin, I was testing out anvil combining mechanics, wanting to see among other things how much a repair would cost past the vanilla limit, since I use another popular plugin (AuraSkills) that allows users to bypass that limit. That's where I ran into this issue. I hope you understand.
